### DISCUSSION TITLE
search.naver.com

### DISCUSSION THREAD

**Discussion by relaxedbrownie** [List-KR/community#59] [Q&A]
https://github.com/List-KR/community/assets/126370491/56ef51e1-025e-4f64-8dd1-c77c086b2060

어떤 사용자 규칙을 넣어야 접속이 가능하도록 바꿀 수 있나요? 영상은 아무거나 하나만 찍어본 거고 다른 검색어를 통해 나오는 광고 배너도 다 차단되네요. 원래 adcr.naver.com을 예외 처리해서 잘 접속했었는데 언젠가부터 갑자기 이게 안 먹히네요.

베이스 필터랑 list-kr 사용 중이에요

**Comment by piquark6046** [comment]
Chromium 브라우저에서는 `adcr.naver.com`이 맞는데요?
Safari 보호 -> 사용자 규칙에 `@@||adcr.naver.com^`를 추가해보세요.

**Reply by relaxedbrownie** [reply]
> Chromium 브라우저에서는 `adcr.naver.com`이 맞는데요? Safari 보호 -> 사용자 규칙에 `@@||adcr.naver.com^`를 추가해보세요.

이거 예전부터 해봤는데 아예 안 먹혀요. 원래 먹혔는데 갑자기 안 먹히네요. 다른 도메인들은 잘 적용되는데 이것만 그러네요.

일단 Safari 사용자 규칙에 `m.search.naver.com#@#section[class*="ad_"]`를 넣어서 제가 들어가려는 광고 배너가 나올 수 있도록 했고요. 그 다음에 `@@||adcr.naver.com^`를 넣으면 원래 들어가졌었거든요? 근데 약 1~2달 전부터 계속 차단돼요. `@@||adcr.naver.com^$important`를 넣어도 마찬가지고 허용 목록에 `adcr.naver.com`를 넣어도 소용없어요. 확인해주실 수 있을까요?

**Reply by piquark6046** [reply]
`er.search.naver.com` 도메인도 예외처리해보세요.

**Reply by relaxedbrownie** [reply]
> `er.search.naver.com` 도메인도 예외처리해보세요.

해봤는데 여전히 차단돼요 ㅠㅠ
![IMG_6766](https://github.com/List-KR/community/assets/126370491/c5b9f921-3e26-4802-83d1-f6d098a056d5)

**Reply by piquark6046** [reply]
https://github.com/AdguardTeam/AdguardForiOS/issues/2315 이 문제 같기도 합니다.

**Reply by relaxedbrownie** [reply]
그러면 지금 당장은 List-KR 안 끄고 예외 처리로 허용할 방법은 없는 거죠?

**Reply by piquark6046** [reply]
네. 안타깝게도 그렇습니다.
아니면, Android 디바이스로 변경하셔서 할 수 밖에 없습니다.
